src/CSS/Components.css

```diff
@@ -158,6 +204,17 @@ nav{
   }
 
   /*state changes*/
+  .heart {
+    transition: 0.2s linear;
+    background: none;
+    border: none;
+  }
+
+  .heart:hover {
+    transform: scale(1.2);
+    cursor: pointer;
+  }
+
   .active {
     text-decoration: underline;
     text-decoration-color: var(--main-red);
```

Comment:
Some seriously impressive CSS in this repo Aaron - loving the hover animation on articles and the hearts for votes is a really nice touch.